Sources/Commands/APIDigester.swift

```diff
@@ -112,6 +113,7 @@ struct APIDigesterBaselineDumper {
             buildParameters: buildParameters,
             cacheBuildManifest: false,
             packageGraphLoader: { graph },
+            verbosity: TSCUtility.verbosity,
```

Comment:
should we define a verbosity variable and share it between test two calls?

Reply:
This is how it behaves until now anyway, doesn't it? I mean, the global verbosity is what is used.